packages/next-on-pages/src/buildApplication/generateFunctionsMap.ts

```diff
@@ -703,3 +823,22 @@ async function copyWasmFiles(
 		await copyFile(originalFileLocation, newLocation);
 	}
 }
+
+/**
+ * Creates js files into the __next-on-pages-dist__/nextjs-manifest folder which export a default
+ * value containing a Next.js manifest object
+ *
+ * @param distDir the __next-on-pages-dist__ directory's path
+ * @param manifests map mapping next manifest identifiers to the manifest objects
+ */
+async function createNextJsManifestFiles(
+	distDir: string,
+	manifests: Map<string, string>
+): Promise<void> {
+	const manifestsDistDir = join(distDir, 'nextjs-manifests');
+	await mkdir(manifestsDistDir);
```

Comment:
Do you need to add `{recursive: true}` here to ensure that parent directories are created if missing?

Reply:
This code only runs after the build process which creates the parent directories.

The parent directories might not be generated (for example if the whole output is static) but if they are not then as a result no manifest is collected and this code doesn't run, so in any case it should be all good.

But do you think we should add `{ recursive: true }` just in case?